eZ/Publish/Core/SignalSlot/ContentService.php

```diff
@@ -650,8 +650,8 @@ public function removeTranslation(ContentInfo $contentInfo, $languageCode)
      *
      * NOTE: this operation is risky and permanent, so user interface should provide a warning before performing it.
      *
-     * @throws \eZ\Publish\API\Repository\Exceptions\BadStateException if the specified translation
-     *         is the only one a Version has or it is the main translation of a Content Object.
+     * @throws \eZ\Publish\API\Repository\Exceptions\BadStateException if the specified Translation
+     *         is the Main Translation of a Content Object.
```

Comment:
"of the content item" would be closer from our domain.

Reply:
> "of the content item" would be closer from our domain.

Amended in e6e674d, hope you won't mind keeping capital letters on domain keywords.